bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieJournalTest.java

```diff
@@ -640,6 +637,15 @@ public void testTruncatedInEntryJournal() throws Exception {
         }
     }
 
+    private Bookie startBookieReadJournal(ServerConfiguration conf) throws IOException, InterruptedException, BookieException {
```

Comment:
nit: technically you are not starting Bookie (not calling start method of Bookie), so better rename it to createBookieAndReadJournal.